includes/settings/js/src/components/fields/MultipleChoiceFields.jsx

```diff
@@ -40,17 +54,17 @@ function MultipleChoiceFields({
 								return (
 									<div
 										key={item.id}
-										className="field multiple-option-container-single"
+										className={`choices[${index}].container field multiple-option-container-single`}
```

Comment:
That's an interesting class name. Do we want '.'s in the class name? Like would this be something like:
choices[1].container field ...

Or is this a WP standard thing?

Reply:
it surprisingly works, but yeah I think we could switch these to `-` between each word. `choices-1-container`

Reply:
If we're using that class only as a selector for the “remove choice” e2e test, another option is to:

1. Remove the className here:

```suggestion
										className={`field multiple-option-container-single`}
```
2. Change the test selector:

```diff
diff --git a/tests/acceptance/CreateContentModelMultipleChoiceFieldCest.php b/tests/acceptance/CreateContentModelMultipleChoiceFieldCest.php
index 3fc3613..26fe3f3 100644
--- a/tests/acceptance/CreateContentModelMultipleChoiceFieldCest.php
+++ b/tests/acceptance/CreateContentModelMultipleChoiceFieldCest.php
@@ -28,7 +28,7 @@ class CreateContentModelMultipleChoiceFieldCest
         $I->wait(1);
         $I->seeInField('choices[0].name', 'dog');
         $I->see('Editing “Favorite Animal” Field', '.field-list');
-        $I->click('.remove-option', '.choices[0].container');
+        $I->click('Remove choice');
         $I->wait(1);
         $I->seeInField('choices[0].name', 'cat');
     }
```

The test runner will click the first button with the “Remove choice” text, which is the same button that was previously identified by `choices[0].container`.